01-Login/webappexample/settings.py

```diff
@@ -136,7 +136,9 @@
     'profile'
 ]
 SOCIAL_AUTH_AUTH0_DOMAIN = os.environ.get('AUTH0_DOMAIN')
-AUDIENCE = os.environ.get('AUTH0_AUDIENCE')
+AUDIENCE = None
+if SOCIAL_AUTH_AUTH0_DOMAIN:
+    AUDIENCE = os.environ.get('AUTH0_AUDIENCE', 'https://' + SOCIAL_AUTH_AUTH0_DOMAIN + '/userinfo')
 if AUDIENCE:
```

Comment:
This should be:

-if os.environ.get('AUTH0_AUDIENCE') has a non empty value, use that one, if not, default to domain + userinfo.

Now it will always override it with domain + userinfo.